### PR TITLE
Options page search link updates

### PR DESCRIPTION
**Pull request by mikehuston**
Makes the following updates to the options page:

* removes search link in top right of page that was not fully implemented
* updates styling of search button on bottom of page to be consistent with rest of page

/cc @zendesk/volunteer 

Before:
screen shot 2016-11-30 at 5 42 25 pm

After (updated):
screen shot 2016-12-01 at 11 29 59 am

### CODE REVIEW

File: pages/options/options.css [hunk 1]
```diff
@@ -94,18 +94,17 @@
 }
 
 .searchButton {
-  background-color: #F3F3F3;
-  background-image: linear-gradient(white,#EBEBEB);
+  background-color: #FFFFFF;
   border: 1px solid #DCDCDC;
   border-radius: 13px;
```
Comment (thebritican) [inline pages/options/options.css]: can we tone down this border radius?

Comment (roseleaf) [inline pages/options/options.css]: what he said

Comment (mikehuston) [inline pages/options/options.css]: Done! Good call

**Comment by mikehuston** [comment]
:haircut_man: @thebritican @roseleaf

**Pull request merged**